src/pages/explore/[[...slug]].js

```diff
@@ -101,6 +106,8 @@ export async function getStaticProps({ preview, previewData, params }) {
       locationCodes,
       profile,
       variant: "explore",
+      preferredChildren: configuration?.preferred_children,
+      featuredGeographies: configuration?.featured_geographies,
```

Comment:
@kilemensi it could, but I did see the need to add it there. Profile gets updated with every router push. And this values are constant, no point of hitting `profile_by_url` several times for the same data

Reply:
So `all_details` endpoint doesn't return all the details then @KhadijaMahanga ? i.e. we can only get the configuration information using `profile_by_url`, right?

Reply:
@kilemensi yes